src/common/pagination.py

```diff
@@ -36,17 +36,19 @@ class EncryptionClient:
     cache: aws_encryption_sdk.caches.base.CryptoMaterialsCache
     materials_manager: aws_encryption_sdk.materials_managers.base.CryptoMaterialsManager
 
-    def encrypt(self, plaintext: Union[str, bytes]) -> bytes:
+    def encrypt(self, plaintext: Union[str, bytes], encryption_context: Union[None, dict]=None) -> bytes:
```

Comment:
Prefer `Optional[T]` to `Union[None, T]`; they are equivalent but `Optional` is a little clearer about intent. It's murkier when you already have a union, but people seem to prefer `Optional[Union[T1, T2]]` to `Union[None, T1, T2]` in [this poll I ran](https://twitter.com/ben11kehoe/status/1484557670463905797?s=20).